cmd/eksctl-anywhere/cmd/listpackages.go

```diff
@@ -21,13 +21,17 @@ var lpo = &listPackagesOption{}
 
 func init() {
 	listCmd.AddCommand(listPackagesCommand)
-	listPackagesCommand.Flags().Var(&lpo.source, "source", "Discovery Location. Options (cluster, registry)")
-	err := listPackagesCommand.MarkFlagRequired("source")
-	if err != nil {
-		log.Fatalf("Error marking flag as required: %v", err)
+
+	listPackagesCommand.Flags().Var(&lpo.source, "source",
+		"Packages info discovery source. Options: cluster, registry.")
```

Comment:
Maybe "Packages discovery source"

Reply:
I went back and forth on that, but landed on using info, since we're just getting package information, not the packages themselves. *shrug*

Reply:
I try to avoid using informal words like "info" in documentation, but I am not strongly opposed either way